openghg/standardise/flux_timeseries/_crf.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from pathlib import Path
+from typing import Dict, Optional, Union
+from openghg.store import infer_date_range
+
+
+def parse_crf(
+    filepath: Path,
+    species: str,
+    source: str = "anthro",
+    domain: Optional[str] = None,
+    data_type: str = "flux_timeseries",
+    database: Optional[str] = None,
+    database_version: Optional[str] = None,
+    model: Optional[str] = None,
+    region: str = "UK",
```

Comment:
Similiarly, would be good to make sure `region` is above `domain` in the function call to emphasise it's use as an identifying key.